backend/src/main/java/ca/bc/gov/app/controller/ches/ChesController.java

```diff
@@ -0,0 +1,21 @@
+package ca.bc.gov.app.controller.ches;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import ca.bc.gov.app.dto.client.EmailRequestDto;
+import ca.bc.gov.app.service.client.ClientService;
+import reactor.core.publisher.Mono;
+
+public class ChesController {
```

Comment:
Sure! I will do that

Reply:
BTW, I just put the RestController annotation, but I don't need to use the @RequestMapping annotation because the @PostMapping annotation already serves the purpose of mapping the sendEmail method to a specific URL and HTTP method